src/main/java/org/gridsuite/explore/server/services/IDirectoryElementsService.java

```diff
@@ -32,7 +32,8 @@ default List<ElementAttributes> completeElementAttribute(List<ElementAttributes>
         Map<String, ElementAttributes> mapElementAttribute = lstElementAttribute.stream()
                 .collect(Collectors.toMap(e -> e.getElementUuid().toString(), Function.identity()));
         /* getting metadata from services */
-        List<Map<String, Object>> metadata = getMetadata(lstElementAttribute.stream().map(ElementAttributes::getElementUuid).collect(Collectors.toList()));
+        List<UUID> collect = lstElementAttribute.stream().map(ElementAttributes::getElementUuid).collect(Collectors.toList());
+        List<Map<String, Object>> metadata = collect.stream().map(e -> Map.of("id", (Object) e)).collect(Collectors.toList());
```

Comment:
why the call of getMetadata method has disappeared ?

Reply:
fixed